support/validation/x86_64-linux-kernel2/Vagrantfile

```diff
@@ -9,6 +9,21 @@ Vagrant.configure("2") do |config|
   end
 
   config.vm.provision "shell", inline: <<-SHELL
-    yum install -y wget curl 
+    yum install -y wget curl git
+
+    # You'll need this for running most services
+    adduser hab
+
+    # You'll want this if you want to build a plan
+    git clone https://github.com/habitat-sh/core-plans.git
+
+    # You'll also want the keys for building for whatever origin you
+    # choose.
+    #
+    # It expects you to have HAB_ORIGIN and HAB_AUTH_TOKEN set in your
+    # environment when you provision the VM.
+    curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh -o install.sh
+    bash install.sh -t x86_64-linux-kernel2
+    sudo -u vagrant hab origin key download #{ENV['HAB_ORIGIN']} --secret --auth=#{ENV['HAB_AUTH_TOKEN']}
```

Comment:
Rather than use a real key, would it make sense to set `HAB_ORIGIN` to something obviously intended for validation, and then generate a key for that origin?   In the context of validating a release, would we ever expect to upload the package we just built? 

Now that I've written that, I think yes we would want upload the package (to acceptance probably) as part of validating a release.   Going to go ahead and hit enter and see what you think.

Reply:
I really like the idea of having a special origin for packages that are just for testing and validation... I know I've written a few of those, and I'm sure you have, too. It'd be nice to formalize that, share the packages, and also incorporate them into our broader testing initiatives and practices.

I was looking for an easy way to get up and running with something, and this seemed the most straightforward. If / when we have a validation origin, we could tweak this, or run the provisioning with a helper script / makefile target that set `HAB_ORIGIN` to that validation origin.

(I don't think that would need to block this, though.)